pkgs/top-level/python-packages.nix

```diff
@@ -11612,7 +11613,7 @@ in {
     propagatedBuildInputs = with self; [
       pytz six tzlocal keyring argparse dateutil_1_5
       parsedatetime
-    ];
+    ] ++ optional (supportEncryption) [ pycrypto ];
```

Comment:
I think you can even just skip the option. It can be easily disabled by using `overrideDerivation`.